promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,105 @@
+plugins that allow you to query and visualize data from a compatible database.
+To add a data source in Grafana you must be signed as a user with organization 
+administration role privileges.
+
+To connect Grafana with Promscale, install and run Grafana version 5.3 or later.
+For information about installing Grafana, see the
+[Grafana installation documentation][grafana-install].
+
+This section shows you how to connect Promscale to Prometheus, Jaeger, and
+PostgreSQL data sources in [Grafana][grafana-homepage].
+
+<procedure>
+
+## Configuring Promscale as Prometheus data source
+
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Prometheus` in the search field and click `Select`.
+1.  Configure the data source:
+    *   In the `Name`field, type `Promscale-metrics`.
+    *   In the `URL` field, type `http://<PROMSCALE-IP-ADDR>:9201`, where
+        `<PROMSCALE-IP-ADDR>` is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+When you have configured Promscale as a Prometheus data source in Grafana, you
+can create panels that are populated with data using PromQL.
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/images/misc/getting-started-with-promscale-grafana-dashboard.png"
+alt="Sample output for PromQl query"/>
+
+Grafana ships with built-in support for Jaeger, which provides open source,
+end-to-end distributed tracing.
+
+<procedure>
+
+## Configuring Promscale as Jaeger data source
+
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Jaeger` in the search field and click `Select`.
+1.  Configure the data source settings:
+    *   In the `Name`field, type `Promscale-traces`.
+    *   In the `URL`field, type `http://<PROMSCALE-IP-ADDR>:9201`, where
+        `<PROMSCALE-IP-ADDR>`is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+You can now filter and view traces stored in Promscale using Grafana. To
+visualize your traces, click the **Explore** button to see the traces filtering
+panel.
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/images/misc/grafana-jaeger-query-results.png"
+alt="Sample output for Jaeger query filter in Grafana"/>
+
+<procedure>
+
+To configure Promscale as PostgreSQL data source you need details such as host,
+port, database, user, and password. These are available in the PostgreSQL service. The format of that URL is `postgresql://[user[:password]@][host][:port][/dbname][?param1=value1&...]`
```

Comment:
```suggestion
port, database, user, and password of the underlying TimescaleDB / PostgreSQL database used by Promscale.
```
There is no such thing as PostgreSQL service (service is a Timescale Cloud name) and the parameters are when you start the database are not passed like in that URL.I think we are listing the url because this is how we show it in Timescale Cloud and we had it in our blog. I think we can just remove this whole sentence.